analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.cs

```diff
@@ -1,9 +1,5 @@
 ﻿using System;
 using System.Collections;
-using System.Collections.Generic;
```

Comment:
https://github.com/SonarSource/sonar-dotnet/pull/8093#discussion_r1342448835

All unneeded but `System` (for `DateTime`) and `System.Collections`.
Fixed.